server/controllers/buyer/addUserOrdersById.js

```diff
@@ -0,0 +1,29 @@
+const { throwError } = require('../../utilities');
+const { addOrdersById } = require('../../database/queries');
+
+const addUserOrdersById = (req, res, next) => {
+  const {
+    productId,
+    quantity,
+    price,
+    paymentMethod,
+  } = req.body;
+  const { userId } = req.params;
+
+  if (quantity < 1) {
+    throw throwError(400, 'quantity must be greater than 1');
+  } else {
+    const orderNumber = `${Date.now()}${productId}‏${userId}`;
+    addOrdersById(
+      productId,
+      quantity,
+      price,
```

Comment:
no need to send the price to the server, it's good to make a query to the database and get the price from here.